reference/library/src/morpheus-master/bootstrap-sass-3.3.6/assets/stylesheets/bootstrap/_scaffolding.scss

```diff
@@ -75,6 +75,8 @@ figure {
 
 img {
   vertical-align: middle;
+  max-width: 100%;
```

Comment:
Could we override this, rather than customising bootstrap itself, otherwise when upgrading to a newer version of bootstrap we'll lose it.